rct229/ruletest_engine/ruletest_engine.py

```diff
@@ -337,6 +343,208 @@ def run_section_tests(test_json_name: str, ruleset_doc: str):
     return all_tests_pass
 
 
+def generate_software_test_report(ruleset, section_list, output_json_path):
+    """Runs list of rule test JSONs and aggregates them into a ashrae901_2019_detail_report
+
+    Parameters
+    ----------
+    ruleset: string
+
+        Name of the ruleset (e.g., 'ashrae9012019')
+
+    section_list : list
+
+        List of test JSON directorys in 'test_jsons/[MY_STANDARD]' directory. (e.g., ['section5', 'section6'])
+
+    output_json_path: str
+
+        output_dir: str - directory in which you want the ashrae901_2019_software_testing_report.json to appear
+
+
+    """
+
+    # Initialize report dictionary in which to
```

Comment:
Could we have a ```TO-DO``` comment here to add more versions of the ruleset in the future?

Reply:
Sure it's not a problem to add that. If another ruletest is is ran, you'll note that it raises an exception to let you know it hasn't been set up yet.